api/graphql/schema.graphql

```diff
@@ -1466,6 +1477,23 @@ input UpdateEmployeeProfileInput @validator {
   dreamRoleDepartments: DepartmentBelongsToMany
 }
 
+input CreateCommunityInterestInput @validator {
+  userId: UUID! @rename(attribute: "user_id")
+  community: CommunityBelongsTo!
+  workStreams: WorkStreamBelongsToMany
+  jobInterest: Boolean @rename(attribute: "job_interest")
+  trainingInterest: Boolean @rename(attribute: "training_interest")
+  additionalInformation: String @rename(attribute: "additional_information")
+}
+
+input UpdateCommunityInterestInput @validator {
+  community: CommunityBelongsTo
```

Comment:
I'm not sure it makes sense to be able to change the community of an existing CommunityInterest model?

Reply:
Especially if the set of workstreams is different on the new community.